modules/video/include/opencv2/video/tracking.hpp

```diff
@@ -849,6 +849,43 @@ class CV_EXPORTS_W TrackerDaSiamRPN : public Tracker
     //bool update(InputArray image, CV_OUT Rect& boundingBox) CV_OVERRIDE;
 };
 
+/** @brief the Nano tracker is a super lightweight dnn-based general object tracking.
+ *
+ *  Nano tracker is much faster and extremely lightweight due to special model structure, the whole model size is about 1.1 MB.
+ *  Nano tracker needs two models: one for feature extraction (backbone) and the another for localization (neckhead).
+ *  Please download these two onnx models at:https://github.com/HonglinChu/SiamTrackers/tree/master/NanoTrack/models/onnx.
+ *  Original repo is here: https://github.com/HonglinChu/NanoTrack
+ *  Author:HongLinChu, 1628464345@qq.com
+ */
+class CV_EXPORTS_W TrackerNano : public Tracker
+{
+protected:
+    TrackerNano();  // use ::create()
+public:
+    virtual ~TrackerNano() CV_OVERRIDE;
+
+    struct CV_EXPORTS_W_SIMPLE Params
```

Comment:
I propose to name it `DnnParams`. There are other tracker options besides DNN part.

Reply:
I think the `Params` is enough since the `Parmas` is belong to the `TrackerNano` class.
And user can only create `Params` with the following code:
C++:
```
cv::TrackerNano::Params params;
```
Python:
```
nanoTrackParam = cv2.TrackerNano_Params()
```

Reply:
Ok, should I change the `DasiamRPN` and other dnn based trackers from `Params` to `DnnParams`?

Reply:
Lets keep `Params` here (similar to other classes with nested parameters structure)

Reply:
@alalek @zihaomu I propose to expose both DNN parameters and  Tracking parameters themself like this:
```
    struct CV_EXPORTS_W_SIMPLE TrackingParams
    {
        int exemplarSize = 127;
        int instanceSize = 255;
        float windowInfluence = 0.455f;
        float lr = 0.37f;
        float contextAmount = 0.5;
        bool swapRB = true;
        int totalStride = 16;
        float penaltyK = 0.055f;
    };
    struct CV_EXPORTS_W_SIMPLE DnnParams
   {
        CV_WRAP Params();
        CV_PROP_RW std::string backbone;
        CV_PROP_RW std::string neckhead;
        CV_PROP_RW int backend;
        CV_PROP_RW int target;
    };
```

Reply:
Some parameters are not allowed to be changed in `TrackingParams `, like `exemplarSize`, `instanceSize`, otherwise the model will collapse.

I think the `Params` is enough.